league/manager/models.py

```diff
@@ -76,6 +73,18 @@ def matches_team_played(self):
         return self.matches_won + \
                self.matches_draw + \
                self.matches_lost
+    
```

Comment:
Interesting Case: Is that change set up with frontend team? You are introducing breaking change now. Are they aware?

Reply:
I didn't talk with frontend team, but I think its not big deal. It's just a proposition, not the final change.

But as you wish, I can tag them here and let them see the changes and judge for themselves.
@skam34